### PR TITLE
feat: completed image section in Add Excercise Screen

### PR DESCRIPTION
**Pull request by agwanyaseen**
This Pr completes add Images section in Add New Excercise Screen.

Completed MockUp: https://rge.uber.space/#!/screens/457862391 
-

## Please check that the PR fulfills these requirements

- [x] Set a 100 character limit in your editor/IDE to avoid white space diffs in the PR
- [ ] Tests for the changes have been added (for bug fixes / features)
- [] Added yourself to AUTHORS.md

### CODE REVIEW

**Comment by agwanyaseen** [comment]
Hey @rolandgeider,
Can you go through these once and let me know if changes are needed.

**Comment by rolandgeider** [comment]
Sure, I'll take a look later today

File: lib/screens/add_exercise_screen.dart [hunk 1]
```diff
@@ -148,28 +155,51 @@ class _DuplicatesAndVariationsStepContent extends StatelessWidget {
...
-            onChange: (value) => print(value),
-            title: 'Name',
-            isRequired: true,
+    return Column(
+      children: [
+        RichText(
+          text: TextSpan(
+            style: Theme.of(context).textTheme.caption,
+            children: const <TextSpan>[
+              TextSpan(text: 'Images must be compatible with the '),
```
Comment (rolandgeider) [inline lib/screens/add_exercise_screen.dart:173]: Can you mark the strings as translatable? You just need to add the translation keys to `app_en.arb`, the generated files are created on build.

And now that I think about it, I would just make this one sentence, without underscoring the license name

File: lib/widgets/add_exercise/preview_images.dart [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                icon: const Icon(Icons.add),
+                onPressed: () => pickImages(context),
+              ),
+            ),
+          ),
+        )
+      ]),
+    );
+  }
+}
```
Comment (rolandgeider) [inline lib/widgets/add_exercise/preview_images.dart:74]: This widget looks really nice!

File: lib/screens/add_exercise_screen.dart (hunk 1 above)
Comment (agwanyaseen) [reply lib/screens/add_exercise_screen.dart:173]: ok I'll do that

File: lib/widgets/add_exercise/preview_images.dart (hunk 2 above)
Comment (agwanyaseen) [reply lib/widgets/add_exercise/preview_images.dart:74]: Thanks!

File: lib/providers/add_excercise_provider.dart [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  void addExcerciseImages(List<File> excercizes) {
+    _excerciseImages.addAll(excercizes);
+    notifyListeners();
+  }
+
+  void removeExcercise(String path) {
+    final file = _excerciseImages.where((element) => element.path == path).first;
+    _excerciseImages.remove(file);
+    notifyListeners();
+  }
```
[the remark this answers is not in the document]
Comment (agwanyaseen) [reply lib/providers/add_excercise_provider.dart:18]: Can you tell me which provider exactly. Just put a link to that file here.

[the remark this answers is not in the document]
Comment (rolandgeider) [reply lib/providers/add_excercise_provider.dart:18]: https://github.com/wger-project/flutter/blob/master/lib/providers/exercises.dart

[the remark this answers is not in the document]
Comment (agwanyaseen) [reply lib/providers/add_excercise_provider.dart:18]: I did go through the provider, but didn't understand the approach,
I will setup some time to discuss these with you.

[the remark this answers is not in the document]
Comment (rolandgeider) [reply lib/providers/add_excercise_provider.dart:18]: sure, we can discuss this over discord

**Pull request merged**